mailpoet/assets/js/src/automation/editor/index.tsx

```diff
@@ -72,26 +72,45 @@ function updatingActiveWorkflowNotPossible() {
   );
 }
 
+function useConfirmUnsaved(isSaved) {
+  function onUnload(event) {
+    if (!isSaved) {
+      event.returnValue = __('changesNotSaved', 'mailpoet'); // eslint-disable-line no-param-reassign
```

Comment:
This seems to try to fetch a string from the legacy translation system. Could we put here the string value itself? Not sure, if we want the `The changes you made may not be saved` or something else (`...may be lost`?). @NeosinneR could answer that, I guess.

Reply:
If we are sure, that the changes will be lost, I'd be direct about that and not mention just the possibility of losing the data. `There are unsaved changes, that will be lost. Do you want to continue?` @brezocordero @JanJakes

Reply:
I have changed the text although most of the time it will use [the default text from the browser](https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event#compatibility_notes).